py-polars/tests/unit/operations/test_sort.py

```diff
@@ -982,3 +982,27 @@ def test_sorted_flag_concat_unit(unit_descending: bool) -> None:
     out = pl.concat((b, a))
     assert out.to_list() == [3, 2, 1, None]
     assert out.flags["SORTED_DESC"]
+
+
+def test_sort_descending_nulls_last() -> None:
```

Comment:
just fyi, this test passes also with the current released version that has the bug.
It's only present when using more than one sort column

Reply:
@CaselIT You're right, I've expanded the tests to cover that.